sdk/python/feast/infra/materialization/contrib/bytewax/bytewax_materialization_engine.py

```diff
@@ -0,0 +1,397 @@
+                "completions": pods,
+                "parallelism": pods,
+                "completionMode": "Indexed",
+                "template": {
+                    "spec": {
+                        "restartPolicy": "Never",
+                        "subdomain": f"dataflow-{job_id}",
+                        "initContainers": [
+                            {
+                                "command": [
+                                    "sh",
+                                    "-c",
+                                    f'set -ex\n# Generate hostfile.txt.\necho "dataflow-{job_id}-0.dataflow-{job_id}.{namespace}.svc.cluster.local:9999" > /etc/bytewax/hostfile.txt\nreplicas=$(($BYTEWAX_REPLICAS-1))\nx=1\nwhile [ $x -le $replicas ]\ndo\n  echo "dataflow-{job_id}-$x.dataflow-{job_id}.{namespace}.svc.cluster.local:9999" >> /etc/bytewax/hostfile.txt\n  x=$(( $x + 1 ))\ndone',
+                                ],
+                                "env": [
+                                    {
+                                        "name": "BYTEWAX_REPLICAS",
+                                        "value": f"{pods}",
+                                    }
+                                ],
+                                "image": "busybox",
+                                "imagePullPolicy": "Always",
+                                "name": "init-hostfile",
+                                "resources": {},
+                                "securityContext": {
+                                    "allowPrivilegeEscalation": False,
+                                    "capabilities": {
+                                        "add": ["NET_BIND_SERVICE"],
+                                        "drop": ["ALL"],
+                                    },
+                                    "readOnlyRootFilesystem": True,
+                                },
+                                "terminationMessagePath": "/dev/termination-log",
+                                "terminationMessagePolicy": "File",
+                                "volumeMounts": [
+                                    {"mountPath": "/etc/bytewax", "name": "hostfile"},
+                                    {
+                                        "mountPath": "/tmp/bytewax/",
+                                        "name": "python-files",
+                                    },
+                                    {
+                                        "mountPath": "/var/feast/",
+                                        "name": f"feast-{job_id}",
+                                    },
+                                ],
+                            }
+                        ],
+                        "containers": [
+                            {
+                                "command": ["sh", "-c", "sh ./entrypoint.sh"],
+                                "env": job_env,
+                                "image": "bytewax/bytewax-feast:latest",
```

Comment:
I'm guessing that this is https://hub.docker.com/r/bytewax/bytewax-feast - can we add some docs on what dependencies this container comes with and how it's built?

Additionally, I think this should be configurable (so that feast users can potentially supply a customer container with custom online store implementation), with this image being the default.

Reply:
I think having a repo that builds the image makes sense. Having it as part of the main repo may be good too, since that keeps everything in the same place. Doing it in a subsequent PR is fine by my